awxkit/awxkit/cli/resource.py

```diff
@@ -176,8 +176,12 @@ def parse_resource(client, skip_deprecated=False):
                     ]
                 }
                 _filter = 'key, value'
+            try:
+                connection = client.root.connection
```

Comment:
`client.root` isn't guaranteed to exist (certain commands, like `config`, don't actually establish a connection to the API, because they don't actually talk to the API, but instead just print state):

<img width="721" alt="image" src="https://user-images.githubusercontent.com/214912/77800677-b59d5080-704d-11ea-85b2-1fdbf0ce4528.png">

Reply:
(`config` is the only command I know of that behaves this way: https://github.com/ansible/awx/blob/devel/awxkit/awxkit/cli/client.py#L112)